### ISSUE TITLE
[Please re-test] [Fixed in GH-6052] Missing Visual Focus on Firefox & Safari

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#4740]
Are we missing some visual focus on firefox & Safari? 

- For instance, while navigating around in Store Finder... 
Maybe it's how the browser is...

**Comment by developpeurweb** [comment]
**For QA:**

_Please retest:_ We have set a global visible focus for all "focusable" ui elements, basically if you can tab to the element you should see the blue outline.

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).
